velox/functions/prestosql/JsonFunctions.h

```diff
@@ -121,11 +121,12 @@ struct JsonArrayContainsFunction {
   }
 };
 
-template <typename T>
+template <typename TExec>
 struct JsonArrayLengthFunction {
```

Comment:
Better to move this class to lib for the use of both Presto and Spark.
https://github.com/facebookincubator/velox/tree/main/velox/functions/lib